src/main/java/com/aws/greengrass/ipc/IPCEventStreamService.java

```diff
@@ -51,27 +49,26 @@
 
     private RpcServer rpcServer;
 
-    @Inject
-    private Kernel kernel;
+    private final Kernel kernel;
 
-    @Inject
-    private GreengrassCoreIPCService greengrassCoreIPCService;
+    private final GreengrassCoreIPCService greengrassCoreIPCService;
 
-    @Inject
-    private AuthenticationHandler authenticationHandler;
+    private final AuthenticationHandler authenticationHandler;
 
-    @Inject
-    private Configuration config;
+    private final Configuration config;
 
     private SocketOptions socketOptions;
     private EventLoopGroup eventLoopGroup;
 
+    @Inject
     IPCEventStreamService(Kernel kernel,
-                                 GreengrassCoreIPCService greengrassCoreIPCService,
-                                 Configuration config) {
+                          GreengrassCoreIPCService greengrassCoreIPCService,
+                          Configuration config,
+                          AuthenticationHandler authenticationHandler) {
```

Comment:
This changes to constructor injection. It is not really necessary to make the test to pass. But I think constructor injection is a better pattern.